components/util/opts.rs

```diff
@@ -767,7 +768,14 @@ pub fn from_cmdline_args(args: &[String]) -> ArgumentParsingResult {
     // This must happen after setting the default options, since the prefs rely on
     // on the resource path.
     for pref in opt_match.opt_strs("pref").iter() {
-        prefs::set_pref(pref, PrefValue::Boolean(true));
+        let split: Vec<&str> = pref.split('=').collect();
```

Comment:
this should use splitn I guess? since the value itself might have an equal sign in it

Also, it should do it in a way that doesn't crash if the pref isn't formatted correctly